doc/en/logging.rst

```diff
@@ -190,3 +188,50 @@ option names are:
 * ``log_file_level``
 * ``log_file_format``
 * ``log_file_date_format``
+
+
+.. _log_release_notes:
+
+Release notes
+^^^^^^^^^^^^^
+
+This feature was introduced as a drop-in replacement for the `pytest-catchlog
+<https://pypi.org/project/pytest-catchlog/>`_ plugin and they conflict
+with each other. The backward compatibility API with ``pytest-capturelog``
+has been dropped when this feature was introduced, so if for that reason you
+still need ``pytest-catchlog`` you can disable the internal feature by
+adding to your ``pytest.ini``:
+
+.. code-block:: ini
+
+   [pytest]
+       addopts=-p no:logging
+
+
+.. _log_changes_3_4:
+
+Incompatible changes in pytest 3.4
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This feature was introduced in ``3.3`` and some **incompatible changes** have been
+made in ``3.4`` after community feedback:
+
+* Log levels are no longer changed unless explicitly requested by the :confval:`log_level` configuration
+  or ``--log-level`` command-line options. This allows users to configure logger objects themselves.
+* :ref:`Live Logs <live_logs>` is now disabled by default and can be enabled setting the
+  :confval:`log_cli` configuration option to ``true``.
```

Comment:
why didn't you add a command line flag for `log_cli`?  When writing tests or debugging problems I want to temporarily turn on cli logging using a command line flag and not by editing pytest.ini.

Reply:
Nowadays (after pytest 3.0)  we should avoid adding duplicate command-line flags and options because you can use the new `-o` option to change ini options in the command line:

```
pytest -o log_cli=true
```

Reply:
Ahh I forgot that this is possible nowadays. For me it seems that the live logging feature is put in second place compared to the `-s` command line flag for `--capture=no`. I agree that it does not make sense to add duplicate command-line flags for every option but at least for the `log_cli` option an short command line parameter (similar to `-s`) would be nice.

Reply:
Let's add a short option in another PR if we see demand for it.